README.md

```diff
@@ -1 +1,60 @@
 # dm-borrower-frontend
+
+The Borrower Frontend provides a website for Borrower to view and sign their
+mortgage deeds online.
+
+## Contents
+- [Usage](#usage)
+- [Getting Started](#getting-started)
+- [Acceptance Tests](#acceptance-tests)
+
+## Usage
+```
+GET     /health                         -- renders standard Gov UK template
+GET     /searchdeed                     -- borrower views deed landing page
+POST    /searchdeed/search              -- form POST the response is a html page which will display the deed or an error page
+```
+
+Install the requirements
+```
+pip install -r requirements.txt
+pip install -r requirements_test.txt
+```
+
+Optional: export variable for deed-api
+```
+export DEED_API_BASE_HOST=http://localhost:8000
+```
+
+> default is localhost:9030
+
+Run the unit tests
+```
+source tests.sh
+```
+
+Run the app
+```
+python run.py runserver --host 0.0.0.0
+```
+> optional ```--port 9000``` where 9000 is the number of a port you can supply to start the server on.
+
+## Acceptance Tests
```

Comment:
This crap about acceptance tests I have removed in the past, too much detail, a dev just wants to know where the tests are, what language and how to run them. Simplicity! bullet points over paragraphs.